modules/nf-core/universc/mkref/main.nf

```diff
@@ -0,0 +1,42 @@
+process UNIVERSC_MKREF {
```

Comment:
The `universesc mkref` actually uses `cellranger mkref` so it should be removed.